build-system/eslint-rules/query-selector.js

```diff
@@ -0,0 +1,174 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+module.exports = function(context) {
+  function querySelectorCall(node) {
+    const {callee} = node;
+
+    // If it's not a querySelector(All) call, I don't care about it.
+    const {property} = callee;
+    if (property.type !== 'Identifier' ||
+        !property.name.startsWith('querySelector')) {
+      return;
+    }
+
+    if (property.leadingComments) {
+      const ok = property.leadingComments.some(comment => {
+        return comment.value === 'OK'
```

Comment:
missing ;

Reply:
Hah! We don't lint the stuff in `build-system/eslint-rules` :)

Reply:
We probably should, though.